src/components/tile/Tile.js

```diff
@@ -1,9 +1,14 @@
 import React from "react";
 
-export const Tile = () => {
-  return (
-    <div className="tile-container">
-      
-    </div>
-  );
+export const Tile = (obj) => {
+  const keys = Object.keys(obj);
+  const mappingKeys = keys.map((key) => {
+    if (key[0]) {
```

Comment:
I think this check should be more explicit instead of just checking for the first value. 

e.g. 

`if (key === "title")`